src/schemas.tsx

```diff
@@ -0,0 +1,12 @@
+export interface UserData {
+  id: number;
+  name: string;
+  username: string;
+  profilePicture: string;
+  membership: string;
+}
+export interface SectionData {
```

Comment:
❌ 
this is not data, it is an interface/type
so rename it to something like `SectionType`